.circleci/config.yml

```diff
@@ -325,38 +325,32 @@ jobs:
             fi
       - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run:
-          name: 'Generate Version Number'
-          command: echo "export VERSION=$(npm run --silent get-next-version | xargs)" >> $BASH_ENV
-      - run: echo ${VERSION}
+          name: 'Store current version number'
+          command: echo "export PREVIOUS_VERSION=$(npm run --silent tooling -- version current | xargs)" >> $BASH_ENV
```

Comment:
I like how the name is "current version" but then the variable name is `PREVIOUS_VERSION` 😂

I know why this is, but I think you should call this `CURRENT_VERSION` for clarity in the future. What is called `CURRENT_VERSION` should be renamed `UPCOMING_VERSION` or something similar, since technically it has not been published yet